app/elements/expert-signin/expert-signin.html

```diff
@@ -23,7 +23,7 @@
       response="{{_apiUser}}"></gdetracking-account-get>
 
     <google-signin
-      client-id="47242318878-dik3r14d8jc528h1ao35f8ehqa7tmpe1.apps.googleusercontent.com"
+      client-id="66416776373-3k5goi8hn9d5rih68t8km57iliithohb.apps.googleusercontent.com"
```

Comment:
Note to self: will need to add this client id to the backend so that authenticated requests coming from that client-id will work.